### PR TITLE
DIRKRB-749

### PR DESCRIPTION
**Pull request by liujiayi771**
Add change_password command line tool for remote kadmin tool

### CODE REVIEW

File: kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java [hunk 1]
```diff
@@ -103,6 +104,10 @@ public ByteBuffer handleMessage(ByteBuffer receivedMessage,
                 System.out.println("message type: export keytab req");
                 responseMessage = handleExportKeytabReq(localKadmin, fieldInfos);
                 break;
+            case CHANGE_PWD_REQ:
+                System.out.println("message type changePwd req");
```
Comment (plusplusjiajia) [inline kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java]: This message is better to be "message type: change password req".

File: kerby-kerb/kerb-admin/src/main/java/org/apache/kerby/kerberos/kerb/admin/kadmin/remote/request/ChangePasswordRequest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        xdrFieldInfos[1] = new XdrFieldInfo(1, XdrDataType.INTEGER, 2);
+        xdrFieldInfos[2] = new XdrFieldInfo(2, XdrDataType.STRING, getPrincipal());
+        xdrFieldInfos[3] = new XdrFieldInfo(3, XdrDataType.STRING, getNewPassword());
+
+        AdminMessageCode value = new AdminMessageCode(xdrFieldInfos);
+        byte[] encodeBytes;
+        try {
+            encodeBytes = value.encode();
+        } catch (IOException e) {
+            throw new KrbException("Xdr encode error when generate get principals request.", e);
```
Comment (plusplusjiajia) [inline kerby-kerb/kerb-admin/src/main/java/org/apache/kerby/kerberos/kerb/admin/kadmin/remote/request/ChangePasswordRequest.java]: Wrong exception message？

File: kerby-kerb/kerb-admin/src/main/java/org/apache/kerby/kerberos/kerb/admin/kadmin/remote/command/RemoteChangePasswordCommand.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    public RemoteChangePasswordCommand(AdminClient adminClient) {
+        super(adminClient);
+    }
+
+    @Override
+    public void execute(String input) throws KrbException {
+        String[] items = input.split("\\s+");
+
+        if (items.length < 4) {
```
Comment (plusplusjiajia) [inline kerby-kerb/kerb-admin/src/main/java/org/apache/kerby/kerberos/kerb/admin/kadmin/remote/command/RemoteChangePasswordCommand.java]: I think using "items.length != 4" is a better choice.

File: kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java (hunk 1 above)
Comment (liujiayi771) [reply kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java]: Already fixed.

**Comment by plusplusjiajia** [comment]
LGTM.

**Comment by plusplusjiajia** [comment]
@liujiayi771 Thanks for your contribution.

**Pull request merged**